solutii/marius_ilau/icao/to_icao.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# *-* coding: UTF-8 *-*
+
+"""
+Organizaţia Internaţională a Aviaţiei Civile propune un alfabet în care
+fiecărei litere îi este asignat un cuvânt pentru a evita problemele în
+înțelegerea mesajelor critice.
+
+Pentru a se păstra un istoric al conversațiilor s-a decis transcrierea lor
+conform următoarelor reguli:
+    - fiecare cuvânt este scris pe o singură linie
+    - literele din alfabet sunt separate de o virgulă
+
+Următoarea sarcină ți-a fost asignată:
+    Scrie un program care să primească un fișier ce conține mesajul
+    ce trebuie transmis și generează un fișier numit mesaj.icao ce
+    va conține mesajul scris folosin alfabetul ICAO.
+
+Mai jos găsiți un dicționar ce conține o versiune a alfabetului ICAO:
+"""
+
+ICAO = {
+    'a': 'alfa', 'b': 'bravo', 'c': 'charlie', 'd': 'delta', 'e': 'echo',
+    'f': 'foxtrot', 'g': 'golf', 'h': 'hotel', 'i': 'india', 'j': 'juliett',
+    'k': 'kilo', 'l': 'lima', 'm': 'mike', 'n': 'november', 'o': 'oscar',
+    'p': 'papa', 'q': 'quebec', 'r': 'romeo', 's': 'sierra', 't': 'tango',
+    'u': 'uniform', 'v': 'victor', 'w': 'whiskey', 'x': 'x-ray', 'y': 'yankee',
+    'z': 'zulu'
+}
+
+
+def icao(mesaj):
+    """Funcția va primi calea mesajul ce trebuie transmis și
+    va genera un fișier numit mesaj.icao_intrare ce va conține
+    mesajul scris folosind alfabetul ICAO.
+    """
+    fisier = open("mesaj.icao_intrare", "w+")
```

Comment:
Am vazut ca asa face el singur fisier si ma si lasa sa scriu in el. Prima data am incercat cu a+ parca, dar la fiecare rulare se tot adunau chestii in fisier.